### ISSUE TITLE
Use of ollama is not disabled even without --experimental_use_ollama_server option.

### ISSUE THREAD

**Issue by spinachpasta** [akawashiro/jendeley#281]
### Problem

Running `jendeley launch --db jendeley_db.json --port 5001` resulted in the following error and the program quit:

```
[15:28:38.592] ERROR (2915): Failed to check ollama server: FetchError: request to http://localhost:11434/api/generate/ failed, reason: 
[15:28:38.592] FATAL (2915): Ollama server is not available. http://localhost:11434/
[15:28:38.635] FATAL (2915): Please check https://github.com/akawashiro/jendeley/blob/main/README.md to launch ollama server.
[15:28:38.636] FATAL (2915): Or just wait for a while to launch ollama server if you already ran ./run_ollama.sh
```

### How it should behave

Either provide an option to disable use of Ollama or Ollama should be disabled without `--experimental_use_ollama_server` option.

### Environment

- MacBook Pro M2 macOS Monterey, 12.6.2 (21G320)
- npm version: 10.7.0
- node version: v22.2.0
- jendeley verision: 2.5.0

### Full output:

```
[15:28:38.541] INFO (2915): startServer version: 2.5.0
[15:28:38.541] INFO (2915): Sending request to ollama server: http://localhost:11434/
[15:28:38.590] INFO (2915): jendeley backend server is listening on port 5001
[15:28:38.590] INFO (2915): Open http://localhost:5001 with your browser
[15:28:38.592] ERROR (2915): Failed to check ollama server: FetchError: request to http://localhost:11434/api/generate/ failed, reason: 
[15:28:38.592] FATAL (2915): Ollama server is not available. http://localhost:11434/
[15:28:38.635] FATAL (2915): Please check https://github.com/akawashiro/jendeley/blob/main/README.md to launch ollama server.
[15:28:38.636] FATAL (2915): Or just wait for a while to launch ollama server if you already ran ./run_ollama.sh

```

**Comment by akawashiro** [comment]
Ooops, thank you so much. I will look into this issue this evening.

**Comment by akawashiro** [comment]
I'm checking at https://github.com/akawashiro/jendeley/pull/282.

**Comment by akawashiro** [comment]
Sorry to be late. I fixed at 2.5.1 https://www.npmjs.com/package/@a_kawashiro/jendeley/v/2.5.1. Thank you for your report. 🙇
